cmd/telegraf/telegraf.go

```diff
@@ -271,7 +271,7 @@ func reloadLoop(stop chan struct{}, s service.Service) {
 		log.Printf("I! Tags enabled: %s", c.ListTags())
 
 		if *fPidfile != "" {
-			f, err := os.Create(*fPidfile)
+			f, err := os.OpenFile(*fPidfile, os.O_EXCL|os.O_CREATE|os.O_WRONLY, 0666)
```

Comment:
`0666` is what the previous one did too; [that's the behaviour of `Create`](https://golang.org/pkg/os/#Create). But I agree; `644` is better.